core/controller/src/main/scala/whisk/core/controller/Controller.scala

```diff
@@ -151,6 +151,25 @@ class Controller(val instance: ControllerInstanceId,
     }
   }
 
+  /**
+   * Handles GET /numHealthyInvokers URI.
+   *
+   * @return number of healthy invokers
+   */
+  private val internalInvokerHealthCount = {
+    implicit val executionContext = actorSystem.dispatcher
+    (path("numHealthyInvokers") & get) {
+      complete {
+        loadBalancer
+          .invokerHealth()
+          .map(_.foldLeft(0) { (r, i) =>
+            if (i.status == Healthy) { r + 1 } else { r }
```

Comment:
can we do this with a query param on the existing route, or perhaps better as a sub resource in the invoker path?

`/invokers`, `/invokers/healthy`

Reply:
Also: Scala conveniently has a `count` method on its collections, so you can simplify this to:

```scala
loadBalancer.invokerHealth()map(_.count(_._2 == Healthy).toJson)
```

Reply:
as discussed on slack, I am going to change to use the route `/invokers/healthy/count`